SQL/2016-03-15-HelpForServerProcessesManager.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO help (helpID,parentID,hash,topic,content,created,updated)
```

Comment:
This insert should be an update - Stella has already committed the insert statement.